README.md

```diff
@@ -13,11 +13,19 @@
 [![PRs Welcome][prs-badge]][prs]
 
 Atom package to format your JavaScript, Typescript, CSS, Sass, and JSON using [Prettier](https://github.com/prettier/prettier).
-Comes with powerful optional [ESLint](http://eslint.org/) integration.
+Optionally integrates with [ESLint](http://eslint.org/), [EditorConfig](http://editorconfig.org/), and [stylelint](https://stylelint.io/).
+
+## How to use it
+
+After [installation](#installation) you are ready to use prettier in a way that suits your needs
+- Toggle 'format on save' to have prettier format your code upon saving
+- Run the command `Prettier: Format (Ctrl+Alt+F)` to format your file manually
```

Comment:
Let's specify the Mac command as well?

Reply:
is that just `cmd-alt-f`?  the repo only comes with a single keymap definition and I don't have a mac.